java/ql/lib/semmle/code/java/frameworks/apache/IOGenerated.qll

```diff
@@ -1,684 +1,828 @@
+        "org.apache.commons.io;FileUtils;true;writeStringToFile;(File,String);;Argument[0];generated:create-file",
+        "org.apache.commons.io;FileUtils;true;writeStringToFile;(File,String,Charset);;Argument[0];generated:create-file",
+        "org.apache.commons.io;FileUtils;true;writeStringToFile;(File,String,Charset,boolean);;Argument[0];generated:create-file",
+        "org.apache.commons.io;FileUtils;true;writeStringToFile;(File,String,String);;Argument[0];generated:create-file",
+        "org.apache.commons.io;FileUtils;true;writeStringToFile;(File,String,String,boolean);;Argument[0];generated:create-file",
+        "org.apache.commons.io;FileUtils;true;writeStringToFile;(File,String,boolean);;Argument[0];generated:create-file",
+        "org.apache.commons.io;FilenameUtils;true;concat;(String,String);;Argument[0];generated:regex-use[f]",
+        "org.apache.commons.io;FilenameUtils;true;concat;(String,String);;Argument[1];generated:regex-use[f]",
+        "org.apache.commons.io;FilenameUtils;true;equals;(String,String,boolean,IOCase);;Argument[0];generated:regex-use[f]",
+        "org.apache.commons.io;FilenameUtils;true;equals;(String,String,boolean,IOCase);;Argument[1];generated:regex-use[f]",
+        "org.apache.commons.io;FilenameUtils;true;equalsNormalized;(String,String);;Argument[0];generated:regex-use[f]",
+        "org.apache.commons.io;FilenameUtils;true;equalsNormalized;(String,String);;Argument[1];generated:regex-use[f]",
+        "org.apache.commons.io;FilenameUtils;true;equalsNormalizedOnSystem;(String,String);;Argument[0];generated:regex-use[f]",
+        "org.apache.commons.io;FilenameUtils;true;equalsNormalizedOnSystem;(String,String);;Argument[1];generated:regex-use[f]",
+        "org.apache.commons.io;FilenameUtils;true;getFullPath;(String);;Argument[0];generated:regex-use[f]",
+        "org.apache.commons.io;FilenameUtils;true;getFullPathNoEndSeparator;(String);;Argument[0];generated:regex-use[f]",
+        "org.apache.commons.io;FilenameUtils;true;getPath;(String);;Argument[0];generated:regex-use[f]",
+        "org.apache.commons.io;FilenameUtils;true;getPathNoEndSeparator;(String);;Argument[0];generated:regex-use[f]",
+        "org.apache.commons.io;FilenameUtils;true;getPrefix;(String);;Argument[0];generated:regex-use[f]",
+        "org.apache.commons.io;FilenameUtils;true;getPrefixLength;(String);;Argument[0];generated:regex-use[f]",
+        "org.apache.commons.io;FilenameUtils;true;normalize;(String);;Argument[0];generated:regex-use[f]",
+        "org.apache.commons.io;FilenameUtils;true;normalize;(String,boolean);;Argument[0];generated:regex-use[f]",
+        "org.apache.commons.io;FilenameUtils;true;normalizeNoEndSeparator;(String);;Argument[0];generated:regex-use[f]",
+        "org.apache.commons.io;FilenameUtils;true;normalizeNoEndSeparator;(String,boolean);;Argument[0];generated:regex-use[f]",
```

Comment:
None of these are regexes AFAICT

Reply:
The model generator will no longer derive "regex-use" sinks.